File: plsql/CSharp/PlSqlBaseParser.cs
using Antlr4.Runtime;

public abstract class PlSqlBaseParser : Parser
{
    private bool _isVersion10 = false;
    private bool _isVersion12 = true;

    protected PlSqlBaseParser(ITokenStream input)
        : base(input)
    {
    }

    public bool isVersion10() => _isVersion10;

    public bool isVersion12() => _isVersion12;

    public bool setVersion10(bool value) => _isVersion10 = value;

    public bool setVersion12(bool value) => _isVersion12 = value;
}

File: plsql/Java/PlSqlBaseParser.java
import org.antlr.v4.runtime.*;

public abstract class PlSqlBaseParser extends Parser
{
    private boolean _isVersion12 = true;
    private boolean _isVersion10 = true;

    public PlSqlBaseParser(TokenStream input) {
        super(input);
    }

    public boolean isVersion12() {
        return _isVersion12;
    }

    public void setVersion12(boolean value) {
        _isVersion12 = value;
    }

    public boolean isVersion10() {
        return _isVersion10;
    }

    public void setVersion10(boolean value) {
        _isVersion10 = value;
    }
}
